### PR TITLE
Keg based brewing system

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
Adds in a bottler, craftable only. When used on a keg that is able to be bottled will produce bottles, woo
Adds in a craftable for fermentation kegs
Adds [13] new bottle sprites for the brewing system

fermentation kegs take large amounts of water and crops to produce booze that eather can be bottled up for local use, or shipped in cargo for an amount of wealth. Woo

100% related tots fr fr changes
Fixes produce bags not updating its UI when putting things in a smart frige or biogenerator

### CODE REVIEW

**Comment by Noobetski** [comment]
THE MOONSHINERS GANG RISE

**Comment by benj8560** [comment]
tier 2 seems to be working. However kegs no longer fart out their contents when reset...something I sware they where doing at some point.

**Comment by benj8560** [comment]
partly scratch the ''working'' part...seems redcandy wine is only needing grapes and not its other plants it calls for.  sunflowers and harebells. It refuses to take the secondary crops and does not need them to start fermenting.

File: modular_sojourn/brewing.dm [hunk 1]
```diff
@@ -0,0 +1,538 @@
...
+		if(needed_crop)
+			message += "Crop Needed: [needed_crop] x [needed_crop_amount], current amount of [needed_crop] is [stored_crop].\n"
+		if(needed_crop_one)
+			message += "Second Crop Needed: [needed_crop_one] x [needed_crop_one_amount], current amount of [needed_crop_one] is [stored_crop_one].\n"
+		if(needed_crop_two)
+			message += "Third Crop Needed: [needed_crop_two] x [needed_crop_two_amount], current amount of [needed_crop_two] is [stored_crop_two].\n"
+
+		//Chems
+		if(needed_chem_one)
+			message += "Reagent Needed: [get_reagent_name_by_id(needed_chem_one)] in amount of at lest [needed_chem_one_amount].\n"
```
Comment (benj8560) [inline modular_sojourn/brewing.dm:111]:
```suggestion
			message += "Reagent Needed: [get_reagent_name_by_id(needed_chem_one)] in a amount of at least [needed_chem_one_amount].\n"
```
Here come several trilgrish corrections. Will keep em coming if I find more

File: modular_sojourn/brewing.dm [hunk 2]
```diff
@@ -0,0 +1,538 @@
...
+		if(needed_chem_two)
+			message += "Second Reagent Needed: [get_reagent_name_by_id(needed_chem_two)] in amount of at lest [needed_chem_two_amount].\n"
```
Comment (benj8560) [inline modular_sojourn/brewing.dm:113]:
```suggestion
			message += "Second Reagent Needed: [get_reagent_name_by_id(needed_chem_two)] in a amount of at least [needed_chem_two_amount].\n"
```

File: modular_sojourn/brewing.dm [hunk 3]
```diff
@@ -0,0 +1,538 @@
...
+	//Misc after the fact stuff
+	var/brewed_amount							//how many bottles we give
+	var/brew_timer								//How *long* it takes to brew a booze (in seconds)
+	var/price_tag_setter						//For selling pre-brewed kegs
+
+	price_tag = 150
+
+/obj/item/bottle_kit
+	name = "Bottler Kit"
+	desc = "A box that holds glasses fabricators, lables, caps and quarks ready for home brewing ad bottling."
```
Comment (benj8560) [inline modular_sojourn/brewing.dm:38]:
```suggestion
	desc = "A box that holds glasses fabricators, lables, caps and quarks ready for home brewing and bottling."
```

File: modular_sojourn/brewing.dm [hunk 4]
```diff
@@ -0,0 +1,538 @@
...
+	var/needed_chem_two_amount					//a second chemical needed amount
+	//Misc after the fact stuff
+	var/brewed_amount							//how many bottles we give
+	var/brew_timer								//How *long* it takes to brew a booze (in seconds)
+	var/price_tag_setter						//For selling pre-brewed kegs
+
+	price_tag = 150
+
+/obj/item/bottle_kit
+	name = "Bottler Kit"
```
Comment (benj8560) [inline modular_sojourn/brewing.dm:37]:
```suggestion
	name = "bottling Kit"
```
Sounds a lil better is all.

**Comment by benj8560** [comment]
think I got everything. Side note. What is this...''fencer'' icon set you've added? Just highly curious they look uh....well...odd

**Comment by benj8560** [comment]
Code_toc5tpirJY

dis stuff?? What is

**Comment by benj8560** [comment]
bottling on redcandy does not seem to work. Could very likely be related to the prior issue. Still works just fine on normal wine.

**Comment by Trilbyspaceclone** [comment]
> dis stuff?? What is

test sprites and wip stuff i didnt mean to commit in, oh well

**Pull request merged**